docs/v1/use-cases.md

```diff
@@ -1,96 +1,142 @@
-Kubernetes.  This offers a new set of users to the service operator and offers
-a wide variety of SaaS products to users of Kubernetes.
+### Updating a service broker
 
-### Exposing Kubernetes services outside the cluster
+Broker authors make changes to the services their brokers offer. To refresh the
+services a broker offers, the catalog should re-list the `/v2/catalog` endpoint.
+The catalog should apply the result of re-listing the broker to its internal
+representation of that broker's services:
 
-It should be possible for service operators whose services are deployed in a
-Kubernetes cluster to publish their services using a service broker.  This
-would allow these operators to participate in the existing service broker
-ecosystem and grow their user base accordingly.
+1. New service present in the re-list results are added
+2. Existing services are updated if a diff is present
+3. Existing services missing from the re-list are deleted
 
-## CF Service Broker `v2` API Use Cases
+TODO: spell out various update scenarios and how they affect end-users
 
-Initially, the catalog should support the current [CF Service Broker
-API](https://docs.cloudfoundry.org/services/api.html) These are the use cases
-that the service catalog has to implement in order to use that API.
+### Delete a service broker
 
-### Managing service brokers
+There must be a way to delete brokers from the catalog. In Cloud Foundry, it is
+possible to delete a broker and leave orphaned service instances. We should
+evaluate where all broker deletes should:
 
-1.  As user, I want to be able to register a broker with the Kubernetes service
-    catalog, so that the catalog is aware of the services that broker offers
-2.  As a user, I want to be able to update a registered broker so that the
-    catalog can maintain the most recent versions of services that broker offers
-3.  As a user, I want to be able to delete a broker from the catalog, so that I
-    can keep the catalog clean of brokers I no longer want to support
+1. Cascade down to the service instances for the broker
+2. Leave orphaned service instances in the catalog
+3. Fail if service instances still exist for the broker
 
-#### Registering a service broker with the catalog
+## Catalog Publishing/Curation/Discovery
 
-An user must register each service broker with the service catalog to
-advertise the services it offers to the catalog.  After the broker has been
-registered with the catalog, the catalog makes a call to the service broker's
-`/v2/catalog` endpoint.  The broker's returns a list of services offered by
-that broker.  Each Service has a set of plans that differentiate the tiers of
-that service.
+* How are services identified: name, service name/id, plan name/id?
+* Who can see which services? (TODO: Include scope? Global/Namespaced)
+* Who can see which service instances? (TODO: Include scope? Global/Namespaced)
+* Which service instances are globally visible?
```

Comment:
I am not convinced at all that any service instances should be globally visible.  I am starting to think that instances should be namespaced and not visible outside of that namespace.